config.go

```diff
@@ -75,13 +78,23 @@ type Configuration struct {
 	BundleSendTimeout   time.Duration
 	BundleSizeMax       int64
 
+	// Compress data on S3 or file output types
+	FileHandlerCompressData bool
+
 	TLSConfig *tls.Config
 
 	// optional post processing of feed hits to retrieve titles
 	PerformFeedPostprocessing bool
 	CbAPIToken                string
 	CbAPIVerifySSL            bool
 	CbAPIProxyUrl             string
+
+	// Kafka-specific configuration
+	KafkaBrokers     *string
+	KafkaTopicSuffix *string
+
+    //Splunkd
+    SplunkToken *string
```

Comment:
Run go fmt on the code base just to fix up little things like the indentation here